core/modfunc.go

```diff
@@ -137,6 +137,14 @@ func (fn *ModuleFunction) setCallInputs(ctx context.Context, opts *CallOpts) ([]
 			return nil, fmt.Errorf("failed to convert arg %q: %w", input.Name, err)
 		}
 
+		// Disable for now until we figure out how to not "double-ignore" arguments from CLI
+		// if len(arg.metadata.Ignore) > 0 {
+		// 	converted, err = fn.applyIgnoreOnDir(ctx, opts.Server, arg.metadata, converted)
+		// 	if err != nil {
+		// 		return nil, fmt.Errorf("failed to apply ignore pattern on arg %q: %w", input.Name, err)
+		// 	}
+		// }
```

Comment:
Why are we disabling this?

What's the issue with the double-ignore? My expectation is that this would work right? It's just that we would not actually end-up excluding things, so it's slightly worse performance, but it would still do the expected thing right?

If it's just that, we should enable it, leave a note, and come back and optimize later by fixing the "double-ignore" issue.

Reply:
Because of this: https://github.com/dagger/dagger/pull/8436#discussion_r1763662136, but I think I'll just remove that part of the code since I talked with Helder and he said it wasn't very important to support ignore in module to module

Reply:
Removed in [d974dcb](https://github.com/dagger/dagger/pull/8436/commits/d974dcbd73cbb7d23e947bc087577d451a75216e), supporting ignore from module to module will be done in a follow up later.

Reply:
> Why are we disabling this?

The implementation didn't seem to correctly handle the ignore patterns in `/sdk/python/dev` when used as a dependency in `/.dagger` (missing `pyproject.toml` for example). I suggested releasing the CLI use case first, depending on how long it would take to do this properly for all directories, since users are reporting slowness there.